opentelemetry/proto/metrics/v1/metrics.proto

```diff
@@ -28,6 +28,9 @@ option go_package = "github.com/open-telemetry/opentelemetry-proto/gen/go/metric
 message ResourceMetrics {
   // The resource for the metrics in this message.
   // If this field is not set then no resource info is known.
+  // The OpenTelemetry API specification further restricts the allowed value
+  // types for Attributes of the Resource:
+  // https://github.com/open-telemetry/opentelemetry-specification/blob/main/specification/common/common.md#attributes
```

Comment:
Why here and not inside Resource?

Reply:
Good point, I moved the comment there in 39680c since that applies to traces, metrics and logs.
(jumping in for @pirgeo who's on vacation 🙂)